### ISSUE TITLE
Crash during local App Engine: Validation Error: Java App Engine components not installed

### ISSUE THREAD

**Issue by etanshaul** [GoogleCloudPlatform/cloud-code-intellij#2939]
IDEA 2021.1
Plugin version 21.5.1

```
Validation Error: Java App Engine components not installed. Fix by running 'gcloud components install app-engine-java' on command-line.
error.stacktrace
com.google.cloud.tools.appengine.operations.cloudsdk.AppEngineJavaComponentsNotInstalledException: Validation Error: Java App Engine components not installed. Fix by running 'gcloud components install app-engine-java' on command-line.
	at com.google.cloud.tools.appengine.operations.CloudSdk.validateAppEngineJavaComponents(CloudSdk.java:247)
	at com.google.cloud.tools.appengine.operations.DevAppServerRunner.run(DevAppServerRunner.java:67)
	at com.google.cloud.tools.appengine.operations.DevServer.run(DevServer.java:133)
	at com.google.cloud.tools.intellij.appengine.java.cloud.executor.AppEngineStandardRunTask.execute(AppEngineStandardRunTask.java:100)
	at com.google.cloud.tools.intellij.appengine.java.cloud.executor.AppEngineExecutor.run(AppEngineExecutor.java:42)
	at com.google.cloud.tools.intellij.appengine.java.ultimate.server.run.CloudSdkStartupPolicy$1$1.createProcessHandler(CloudSdkStartupPolicy.java:183)
	at com.intellij.javaee.appServers.run.execution.PatchedLocalState.startJ2EEProcess(PatchedLocalState.java:96)
	at com.intellij.javaee.appServers.run.execution.J2EEProcessHandlerWrapper.lambda$new$0(J2EEProcessHandlerWrapper.java:96)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:216)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:27)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.execute(BoundedTaskExecutor.java:195)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:213)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:184)
```

**Comment by peterlin741** [comment]
Still trying to reproduce this

**Comment by matthewmichihara** [comment]
Fixed in 21.10.1
